frontend/src/components/viewers/Tensorboard.tsx

```diff
@@ -56,6 +64,10 @@ class TensorboardViewer extends Viewer<TensorboardViewerProps, TensorboardViewer
     this._checkTensorboardApp();
   }
 
+  public onChangeFunc(e: React.ChangeEvent<{ name?: string; value: unknown }>): void {
```

Comment:
2 notes:
1. recommend using arrow function as field like
```
public onChangeFunc = (e: ...) => {
...
}
```
Arrow functions are bound automatically, then you can omit the `.bind..` part when using it.

2. name the function by what it does makes this more readable. e.g. `selectVersion` or `handleVersionSelect`

Reply:
I will rename the function. As for the arrow function, I didn't use it because when writing tests, I failed to invoke onChange arrow function with enzyme's simulate('change') method, although it works fine in real environment. Maybe we can discuss it later.